packages/terra-site/src/examples/date-picker/DatePickerDefault.jsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 import DatePicker from 'terra-date-picker';
 
-const DatePickerExample = () => (
-  <DatePicker
-    name="date-input-default"
-  />
-);
+class DatePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Selected ISO Date: {this.state.date}</p>
```

Comment:
The actual date value for submission is not the entire ISO date string (e.g.; 2017-07-27T00:00:00-05:00). The submission value is only the date part in the ISO date string (e.g.; 2017-07-27). The value that is submitted is the value attribute in the hidden input (the input with type=hidden). This is just a Date component without the time so it shouldn't display/submit any time related value.

![date submission value](https://user-images.githubusercontent.com/22618655/28724090-89be85c0-737e-11e7-8839-3a4332076315.png)